### PR TITLE
derive debug for some rlua types

### PR DESCRIPTION
**Pull request by yjhmelody**
Maybe the low level API's Debug need we to impl it by hand.

### CODE REVIEW

**Comment by jugglerchris** [comment]
Hi - do you have a need for `Debug` on those types?  I can't quite see the use case.  :-)

**Comment by yjhmelody** [comment]
I think they're useful when debugging.

File: src/ffi.rs [hunk 1]
```diff
@@ -3,27 +3,36 @@
...
+use std::os::raw::{
+    c_char, c_double, c_int, c_longlong, c_uchar, c_uint, c_ulonglong, c_ushort, c_void,
+};
 use std::ptr;
 
 pub type lua_Integer = c_longlong;
 pub type lua_Unsigned = c_ulonglong;
 pub type lua_Number = c_double;
 
+#[derive(Debug)]
```
Comment (jugglerchris) [inline src/ffi.rs:15]: This seems wrong to me - given that `lua_State` is, as far as the Rust type system is concerned, an empty enum so can't have any instances.

**Comment by jugglerchris** [comment]
Ok, I can't see any particular reason not to have the `Debug` derives except on `lua_State`.  I'm happy to merge if that one is removed.
(In future it would help if rustfmt changes are in separate commits, but no big deal here)

File: src/ffi.rs (hunk 1 above)
Comment (yjhmelody) [reply src/ffi.rs:15]: I will remove it.

**Comment by jugglerchris** [comment]
Thank you!

One comment for future - it's best not to force push to branches in review; it makes it hard to check changes made since the last review.

**Pull request merged**